ssas/systems.go

```diff
@@ -45,23 +44,6 @@ func getEnvVars() {
 	CredentialExpiration = time.Duration(expirationDays*24) * time.Hour
 }
 
-/*
-	InitializeSystemModels will call gorm.DB.AutoMigrate() for models associated with systems, and set up foreign key
-	relationships for those models if needed
-*/
-func InitializeSystemModels() *gorm.DB {
```

Comment:
No longer needed since we've migrated to golang migrate https://github.com/golang-migrate/migrate